lms/static/js/financial-assistance/views/financial_assistance_form_view.js

```diff
@@ -9,7 +9,7 @@
             'js/student_account/views/FormView',
             'text!../../../templates/financial-assistance/financial_assessment_form.underscore',
             'text!../../../templates/financial-assistance/financial_assessment_submitted.underscore',
-            'text!templates/student_account/form_field.underscore',
+            'text!lms/djangoapps/student_account/templates/student_account/form_field.underscore',
```

Comment:
@jcdyer The ``text!`` link is resolved by the browser, so it is going to look for a file beneath the static directory. You can see above that it resolves ``js/student_account/views/FormView``, which it can find beneath that path.

The approach we're using now is to have a ``static`` directory beneath your Django app, add a namespace to it (in this case ``student_account``) and then have js and templates directories beneath there. So I'd move your underscore file to:

  lms/djangoapps/static/templates/student_account/form_field.underscore

There's documentation of this approach on the wiki here:

https://openedx.atlassian.net/wiki/display/AC/How+to+add+a+new+feature+to+LMS+or+Studio#HowtoaddanewfeaturetoLMSorStudio-django-appHowtoaddanewDjangoapptotheedXplatform

As an example, here's how we laid out discussions:

https://github.com/edx/edx-platform/tree/master/lms/djangoapps/discussion/static/discussion